src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs

```diff
@@ -75,12 +76,16 @@ private Manifest BuildCore(DocumentBuildParameters parameters)
                     parameters.Files.DefaultBaseDir);
                 if (ShouldTraceIncrementalInfo)
                 {
+                    string configHash = ComputeConfigHash(parameters);
+                    string templateHash = parameters.TemplateManager.GetTemplatesHash();
```

Comment:
1.templatemanager is part of parameters, but the resources are not loaded yet, so we still need to calculate its hash
2. documentBuilder also process per parameter

---
In reply to: [75337615](https://github.com/dotnet/docfx/pull/621#discussion_r75337615) [](ancestors = 75337615)